daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/transaction/PartialTransaction.scala

```diff
@@ -35,6 +37,14 @@ private[lf] object PartialTransaction {
   type Node = Node.GenNode[NodeId, Value.ContractId]
   type LeafNode = Node.LeafOnlyActionNode[Value.ContractId]
 
+  private type TX = GenTransaction[NodeId, Value.ContractId]
+  private type ExerciseNode = Node.NodeExercises[NodeId, Value.ContractId]
+
+  private final case class ConcreteIncompleteTx(
```

Comment:
```suggestion
  private final case class IncompleteTxImpl(
```

Reply:
doesn’t matter much but this is common in our codebase and in Scala in general so I don’t see a reason to diverge from that naming scheme.

Reply:
done